### DISCUSSION TITLE
WalletConnect node_modules files not found

### DISCUSSION THREAD

**Discussion by evankalaw** [wevm/wagmi#477] [Q&A]
Hey there, running into one more thing that I am curious about regarding WalletConnect usage in Wagmi:

I'm getting a ton of warnings when building my application, all some flavor of the following:

```
WARNING in ./node_modules/@walletconnect/window-metadata/dist/cjs/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'my-file-path/node_modules/@walletconnect/window-metadata/src/index.ts' file: Error: ENOENT: no such file or directory, open 'my-file-path/node_modules/@walletconnect/window-metadata/src/index.ts'
```

Seems to be a type error, and I've gone ahead and added @walletconnect/types which seemed to be the issue considering its looking for .ts files, but they're all still there unfortunately. Anyone run into this?

**Comment by evankalaw** [comment]
To be quite frank, I'm getting 60+ warnings and there is a variety of packages that are throwing pretty similar errors (some file is not found): @walletconnect/window-metadata @walletconnect/window-getters @walletconnect/utils  @walletconnect/socket-transport @walletconnect/signer-connection @walletconnect/safe-json @walletconnect/randombytes @walletconnect/jsonrpc-utils @walletconnect/jsonrpc-types @walletconnect/jsonrpc-provider @walletconnect/jsonrpc-http-connection @walletconnect/iso-crypto @walletconnect/ethereum-provider @walletconnect/environment @walletconnect/encoding @walletconnect/crypto @walletconnect/core @walletconnect/browser-utils

**Reply by tmm** [reply]
Try removing `node_modules` and doing a fresh install.
